SDK.markdown

```diff
@@ -7,6 +7,104 @@ sidebar: true
 
 We provide you with a NPM package for Node.js and Browser environments. Please see our [GitHub page for details](https://github.com/kontist/sdk).
 
+## Getting started
+This is a very short step by step guide to help you getting started and fetch a few transactions with our API and SDK in Node.js. You can do much more with it, please refer to the documentation or [get in touch with us](mailto:developer@kontist.com).
+
+### Requirements
+Please have Node.js + NPM installed. This example is using plain JavaScript for simplicity, but you can use TypeScript as well.
```

Comment:
Should we mention here that you can use TypeScript as well? Is it important here?

Reply:
Yeah, I think it makes sense, as this will be one of the first sentences read by potential users it is good to reassure them on that front imo 🙂